ddtrace/contrib/redis/util.py

```diff
@@ -18,9 +18,13 @@
 CMD_MAX_LEN = 1000
 
 
-def _extract_conn_tags(conn_kwargs):
+def _extract_conn_tags(instance):
     """Transform redis conn info into dogtrace metas"""
+    if not hasattr(instance, "connection_pool"):
```

Comment:
Can you explain why this check is necessary? I'm not clear on why this function needs to change for supporting `aioredis`.

Reply:
Not all redis clients declare the `connection_pool` field. In aioredis v1.X.X we instrument [Redis.execute](https://github.com/aio-libs/aioredis-py/blob/v1.3.1/aioredis/commands/__init__.py#L31) and in v2.X.X we instrument [StrictRedis.execute_command](https://github.com/aio-libs/aioredis-py/blob/6612ba716320c43a87b2b6c159a62e8569bda86c/aioredis/client.py#L630). The v1 Redis class does not have a connection_pool attribute while v2 has this field

We already have a `hasattr(instance, "connection_pool")` check on redis instances in [flask_cache/utils.py](https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/contrib/flask_cache/utils.py#L45) so I there should be other redis libraries that do not declare a `connection_pool` field.

Reply:
can you add a comment reflecting this?